frontend/src/layout/navigation-3000/components/TopBar.tsx

```diff
@@ -284,7 +285,7 @@ function Breadcrumb({ breadcrumb, here, isOnboarding }: BreadcrumbProps): JSX.El
         )
     }
 
-    return breadcrumbContent
+    return <ErrorBoundary>{breadcrumbContent}</ErrorBoundary>
```

Comment:
Can we have a sane default for the boundary here? Maybe something that will make us recognize an error occured?

Reply:
i agree, that was my first thought. the error boundary however is defined in the sdk, so [I just created a PR for it here](https://github.com/PostHog/posthog-js/pull/1976)